client/src/index.jsx

```diff
@@ -86,11 +93,75 @@ class Zillow extends React.Component {
         main: [],
         group: []
       },
+      value: '',
       mainPage: true,
       index: {}
     };
+    this.deletePhoto = this.deletePhoto.bind(this);
+    this.closePhotoModal = this.closePhotoModal.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  updatePhoto(e) {
+    var path = window.location.pathname.split('/');
+    var house = path[1];
+    var self = this;
+    var url = this.state.value;
+    $.ajax({
+      method: 'PUT',
+      url: `http://localhost:3002/gallery/${house}/photo/${this.state.photoSelected}/url/${this.state.value}`,
+      success: function(data) {
+        //update gallery
+        $.ajax({
+          method: 'GET',
+          url: `http://localhost:3002/gallery/${house}`,
+          success: function(data) {
+            var result = {
+              main: data[0],
+              group: data.slice(1, 5)
+            };
+            self.setState({
+              houseId: house, 
+              gallery: data, 
+              display: result
+            });
+            self.closePhotoModal();
+          } 
+        });
+      }
+    });
+    this.closePhotoModal();
+  }
+  handleChange(event) {
+    this.setState({value: event.target.value});
```

Comment:
Great work on these methods Bernard. This is a nit pick, and won't improve performance, but you might consider destructuring the event object. Here's an example:

` const { value } = event.target;
   this.setState({ value });
`

Reply:
In my opinion seeming how it is only one item I don't think it needs to be done, if he was setting multiple things on setState most likely.  I am basically just writing this to have something to write on this code review though.